openfisca_aotearoa/variables/social_security/accommodation_supplement.py

```diff
@@ -0,0 +1,69 @@
+from openfisca_aotearoa.entities import Person
+from openfisca_core.model_api import *
+
+
+class social_security__eligible_for_accommodation_supplement(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = "Eligible for Accommodation Supplement"
+    reference = "TODO"
```

Comment:
I would recommend not adding a `reference` at all, or leaving it empty.
As free text is allowed, this will be parsed and exposed to the users. It will also make it harder to automatically detect missing references.

_(repeated below)_